.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false # faster builds
 
 jdk:
-- openjdk8
+- openjdk17
```

Comment:
Why would you upgrade here? Java 1.8 is still supported as LTS and as this is a framework, we should try to stick to the bottom line of compatibility.

Reply:
Thanks for pointing this out 👍 But I stick with [GitHub actions](https://github.com/camunda-community-hub/spring-zeebe/blob/master/.github/workflows/mvn-build.yml#L20) and I see here the java requirement for the 17th version: https://github.com/camunda-community-hub/spring-zeebe/blob/568d70df542202c35d4a89f49c9a7a1bde230d08/test/common/pom.xml#L16

Reply:
I'm not sure about Travis in this project at all 😅 Since there are already Github actions :)

Reply:
Awesome. This requires some additional cleaning I guess...

https://github.com/camunda-community-hub/spring-zeebe/issues/223